File: app/src/main/java/me/chrislane/accudrop/viewmodel/JumpStatsViewModel.java
package me.chrislane.accudrop.viewmodel;

import android.arch.lifecycle.LiveData;
import android.arch.lifecycle.MutableLiveData;
import android.arch.lifecycle.ViewModel;

public class JumpStatsViewModel extends ViewModel {

    private MutableLiveData<Integer> jumpId = new MutableLiveData<>();
    private MutableLiveData<Integer> firstJumpId = new MutableLiveData<>();
    private MutableLiveData<Integer> lastJumpId = new MutableLiveData<>();

    public void setJumpId(int jumpId) {
        this.jumpId.setValue(jumpId);
    }

    public LiveData<Integer> getJumpId() {
        return jumpId;
    }

    public void setFirstJumpId(int firstJumpId) {
        this.firstJumpId.setValue(firstJumpId);
    }

    public void setLastJumpId(int lastJumpId) {
        this.lastJumpId.setValue(lastJumpId);
    }

    public LiveData<Integer> getFirstJumpId() {
        return firstJumpId;
    }

    public LiveData<Integer> getLastJumpId() {
        return lastJumpId;
    }
}

File: app/src/main/java/me/chrislane/accudrop/viewmodel/JumpStatsViewModel.kt
package me.chrislane.accudrop.viewmodel

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class JumpStatsViewModel : ViewModel() {

    private val jumpId = MutableLiveData<Int>()
    private val firstJumpId = MutableLiveData<Int>()
    private val lastJumpId = MutableLiveData<Int>()

    fun setJumpId(jumpId: Int) {
        this.jumpId.value = jumpId
    }

    fun getJumpId(): LiveData<Int> {
        return jumpId
    }

    fun setFirstJumpId(firstJumpId: Int) {
        this.firstJumpId.value = firstJumpId
    }

    fun setLastJumpId(lastJumpId: Int) {
        this.lastJumpId.value = lastJumpId
    }

    fun getFirstJumpId(): LiveData<Int> {
        return firstJumpId
    }

    fun getLastJumpId(): LiveData<Int> {
        return lastJumpId
    }
}
